### PR TITLE
Create a 'How to Triage an Adoptium Release' document

### PR DESCRIPTION
**Pull request by diyorbekibragimov**
Fixes: #5328

### CODE REVIEW

**Comment by smlambert** [comment]
thanks @diyorbekibragimov, are you able to [sign the ECA](https://adoptium.net/docs/eca-sign-off/) so that the eclipsefdn/eca check listed above can pass?

**Comment by diyorbekibragimov** [comment]
@smlambert there was an issue with my account. I was using another email to sign the ECA. I've fixed the issue. Everything should be fine now. I am not sure how I can return the check for this pull request though. I'd appreciate your help!

**Comment by diyorbekibragimov** [comment]
Okay, I've passed the checks. Just needed to re-commit

**Review by llxia — APPROVED** [review]
Thanks @diyorbekibragimov

File: docs/pages/HowToTriageanAdoptiumRelease.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# How to Triage an Adoptium Release
+## Recording
+[AQAvit Triage Lightning Introduction Video](https://www.youtube.com/watch?v=TWD_b8cwIVg&ab_channel=EclipseAdoptium)
+
+## PDF Slides
+[AQAvit Triage Lightning Slides](https://github.com/adoptium/aqa-tests/blob/master/docs/pdf/AQAvitTriageLightningSlides.pdf)
```
Comment (smlambert) [inline docs/pages/HowToTriageanAdoptiumRelease.md:6]:
```suggestion
[AQAvit Triage Lightning Slides](https://github.com/adoptium/aqa-tests/files/15349378/AQAvitTriageLightningSlides.pdf)
```

**Review by smlambert — COMMENTED** [review]
One last update needed, to fix the link to the slides, as the other one gives a 404 error.

**Review by karianna — APPROVED** [review]

**Pull request merged**